.github/workflows/publish.yml

```diff
@@ -1,33 +0,0 @@
-name: publish
-
-on:
-  pull_request:
-    branches: [source]
-  push:
-    branches: [source]
-
-jobs:
-  checks:
-    if: github.event_name != 'push'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '10.x'
-      - name: Test Build
-        run: cd website && yarn install && yarn build
-  gh-release:
```

Comment:
Please only delete this gh-release section.

Reply:
I deleted `gh-release` section and fixed PR name and descriptions accordingly.